SpatialGDK/Source/SpatialGDKEditorCommandlet/Private/Commandlets/CookAndGenerateSchemaCommandlet.cpp

```diff
@@ -103,7 +105,13 @@ int32 UCookAndGenerateSchemaCommandlet::Main(const FString& CmdLineParams)
 	}
 
 	UE_LOG(LogCookAndGenerateSchemaCommandlet, Display, TEXT("Starting Cook Command."));
-	int32 CookResult = Super::Main(CmdLineParams);
+
+	const FString AdditionalCookParam(TEXT(" -cookloadonly"));
```

Comment:
This may speed up schema generation but won't it slow down overall build time as the lack of caching means in subsequent steps the cook will need to be re-ran?

I think this should be optional.

Reply:
You have to rerun the cook anyway at the moment

Reply:
I think the original idea of having this command nicely integrated in a cooking flow has been thrown out at the moment.

Reply:
Hmmm, I believe with `-iterate` you will re-use the previous cooked data from the DDC. If this is the first time cooking this data and we don't cache it then the second cook will have nothing to use right?

Reply:
AFAIK the cooker will always use the DDC. `-iterate` flag will use previously cooked files, which isn't great for schema gen as it wont even load the files it's already cooked.